src/printing.py

```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %d, y is %f, z is %s" %(x, y, z))
 
 # Use the 'format' string method to print the same thing
+print("x is {}".format(x), "y is {},".format(y), "z is {}".format(z))
```

Comment:
you can combine all format statements into one, similar to line 14

print("x is {}, y is {:.2f}, z is \"{}\"".format(x, y, z))